packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -214,21 +215,33 @@ const DefaultSummaryConfiguration: ISummaryConfiguration = {
  * Options for container runtime.
  */
 export interface IContainerRuntimeOptions {
-    // Flag that will generate summaries if connected to a service that supports them.
-    // This defaults to true and must be explicitly set to false to disable.
+    /**
+     * Flag that will generate summaries if connected to a service that supports them.
+     * This defaults to true and must be explicitly set to false to disable.
+     */
     generateSummaries?: boolean;
 
-    // Delay before first attempt to spawn summarizing container
+    /* Delay before first attempt to spawn summarizing container. */
     initialSummarizerDelayMs?: number;
 
-    // Flag that will disable garbage collection if set to true.
+    /* Flag that will disable garbage collection if set to true. */
     disableGC?: boolean;
 
-    // Flag that will bypass optimizations and generate GC data for all nodes irrespective of whether the node
-    // changed or not.
+    /**
+     * Flag representing the document's preference for enabling garbage collection.
+     * This is stored in the summary and unchangeable (for now).
+     * Currently if this is set to false, it will take priority and the document will
+     * never run GC.
+     */
+    documentEnableGC?: boolean;
```

Comment:
please don't use the word document

Reply:
What term should I use here? Not sure the term for the "file" entity- i.e. the specific op log + summaries.

Reply:
Changed to `summaryEnableGC` for now.. See other comment about changing again to something else.. like `summaryAllowGC` or otherwise.